pyicloud/services/drive.py

```diff
@@ -97,7 +97,7 @@ def _get_upload_contentws_url(self, file_object):
             return None
         return (request.json()[0]["document_id"], request.json()[0]["url"])
 
-    def _update_contentws(self, folder_id, sf_info, document_id, file_object):
+    def _update_contentws(self, folder_id, sf_info, document_id, file_object,zone='com.apple.CloudDocs'):
```

Comment:
same. Generally speaking, it's a good idea to send your code through black before committing.